### ISSUE TITLE
The "Installing the Rails Gem for Local Users" section needs to be updated

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/rails-application-basics-readme#18]
Hi. The "Installing the Rails Gem for Local Users" section is no longer accurate about the Learn Sandbox.

It says, "If you are using the IDE you should not do this step. You already have rails installed. Just skip down to generating a new rails application!"

However, when I followed the instructions in the in-browser Sandbox by running `rails new blog-flash`, I didn't see any blog-flash directory show up in my file structure, despite the fact that the Sandbox had the Rails gem installed. And when I ran `cd blog-flash`, I got this error:
```
bash: cd: blog-flash: No such file or directory
```
This happened in the downloadable Learn IDE as well.

I asked a Technical Coach about this, and I was informed that commands like `rails new` won't work in the Sandbox, which is why they encourage students to use a local environment instead.

Long story short, the only way to do this lesson correctly is by using a local environment instead of the Learn Sandbox. Because of this, the README needs to be updated.

Thanks as always for looking into this!

Regards,
Sdcrouse

**Comment by sgharms** [comment]
Wow! Rails in the sandbox! You're brave!

Will update.

**Comment by Sdcrouse** [comment]
Thanks! It was more for the sake of curiosity than anything else.

By the way, for future reference, Rails now has a version 6. That doesn't work well with the Learn labs, though.
